lib/iex/lib/iex/h_elixir.ex

```diff
@@ -0,0 +1,143 @@
+defmodule IEx.H_Elixir do
+  @moduledoc """
+  This module duplicates the prior behaviour of the iex h
+  command using the documentation interface.
+
+  This interface consists of the functions
+
+  * `documentation(module)`
+
+  * `documentation(module, function)`
+
+  * `documentation(module, function, arity)`
+
+  All of these functions return a tuple of status and doc_list.
+
+  * `{:found, doc_list}`     - Documentation was found
+
+  * `{:not_found, doc_list}` - Documentation was not found and
+                              this helper is sure it can't be found
+
+  * `{:unknown, doc_list}`   - Helper does not know how to find docs for
+                               arguments given.
+
```

Comment:
yes, sorry about that one. It's used both ways. I just thought it could break the list, but it doesn't. so you are fine.